src/Http/Event/ImportEventRequestHandler.php

```diff
@@ -270,7 +273,17 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         // which might cause race conditions if we're still dispatching other commands here as well.
         $organizerId = $eventAdapter->getOrganizerId();
         if ($organizerId) {
-            $commands[] = new UpdateOrganizer($eventId, $organizerId);
+            try {
+                $this->organizerDocumentRepository->fetch($organizerId);
+                $commands[] = new UpdateOrganizer($eventId, $organizerId);
+            } catch (DocumentDoesNotExist $e) {
```

Comment:
I think that the `DocumentDoesNotExist` exception gets converted to a not found and then a 404 gets returned which is also fine in this context. So I don't think it's needed to catch the `DocumentDoesNotExist` and then convert it to `bodyInvalidData`

Reply:
I gave this a quick try, but the resulting error is not so clear because it uses a general message `Document with id ... not found.`

Reply:
Maybe to make it more readable put this in a `gaurdOrganization` method?